Mathlib/Order/OrderType.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) _
+-/
+import Mathlib.SetTheory.Cardinal.Basic
+import Mathlib.SetTheory.Ordinal.Basic
+import Mathlib.Order.Category.LinOrd
+import Mathlib.Algebra.Order.Ring.Unbundled.Rat
+import Mathlib.Data.Real.Basic
```

Comment:
This shouldn't import real numbers; we can have a separate file where we define the order type of `ℝ` and prove all of the fancy stuff about η-orders.